documentation/src/docs/asciidoc/release-notes/release-notes-5.8.0-M1.adoc

```diff
@@ -31,6 +31,9 @@ on GitHub.
   `test(displayName("my test"))` can be used to match against a test whose display name is
   `my test`.
 
+* New `assertInstanceOf` methods as a replacement for `assertTrue(obj instanceof X)` which
+  provide better error messages comparable to those of `assertThrows`.
+
```

Comment:
This should go to the "New Features and Improvements" section of "JUnit Jupiter"

Reply:
done